src/posix/handlers/stat.hpp

```diff
@@ -112,7 +112,7 @@ inline int capio_fstatat(int dirfd, std::string *pathname, struct stat *statbuf,
             std::string path;
             if (pathname->at(pathname->length() - 1) == '.') {
                 path = dir_path;
-            } else {
+            } else if (pathname[0] != "/") {
```

Comment:
If you want to check if the path is absolute, use the `is_absolute` function for clarity and maintainability.